applications/poisson/overset_grids/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+TARGETNAME(TARGET_NAME ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Why is here no header?

Reply:
I think we generally do not have them for these files. I could include them consistently in a separate PR.